[2018-07-16 12:45:55] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Cannot declare class Protocol\Service\ServiceFactory, because the name is already in use in /var/www/src/App/Service/ServiceFactory.php:20
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 64): Cannot declare class Protocol\\Service\\ServiceFactory, because the name is already in use at /var/www/src/App/Service/ServiceFactory.php:20)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:46:35] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Cannot declare class Protocol\Service\ServiceFactory, because the name is already in use in /var/www/src/App/Service/ServiceFactory.php:20
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 64): Cannot declare class Protocol\\Service\\ServiceFactory, because the name is already in use at /var/www/src/App/Service/ServiceFactory.php:20)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:46:37] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Cannot declare class Protocol\Service\ServiceFactory, because the name is already in use in /var/www/src/App/Service/ServiceFactory.php:20
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 64): Cannot declare class Protocol\\Service\\ServiceFactory, because the name is already in use at /var/www/src/App/Service/ServiceFactory.php:20)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:46:37] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Cannot declare class Protocol\Service\ServiceFactory, because the name is already in use in /var/www/src/App/Service/ServiceFactory.php:20
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 64): Cannot declare class Protocol\\Service\\ServiceFactory, because the name is already in use at /var/www/src/App/Service/ServiceFactory.php:20)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:47:50] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:47:50] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\Lumen\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(ErrorException))
#2 {main}
  thrown in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\\Lumen\\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(ErrorException))
#2 {main}
  thrown at /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:48:01] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:48:01] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\Lumen\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(ErrorException))
#2 {main}
  thrown in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\\Lumen\\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(ErrorException))
#2 {main}
  thrown at /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:48:16] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:48:16] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\Lumen\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(ErrorException))
#2 {main}
  thrown in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\\Lumen\\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(ErrorException))
#2 {main}
  thrown at /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:50:26] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:50:26] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:128
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\Lumen\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(ErrorException))
#2 {main}
  thrown in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:128
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:128
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\\Lumen\\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(ErrorException))
#2 {main}
  thrown at /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:128)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:50:39] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:52:46] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:52:46] production.ERROR: Symfony\Component\Debug\Exception\FatalErrorException: Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\Lumen\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(ErrorException))
#2 {main}
  thrown in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Uncaught Error: Call to a member function send() on null in /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(50): Laravel\\Lumen\\Application->handleUncaughtException(Object(ErrorException))
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(ErrorException))
#2 {main}
  thrown at /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php:127)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(54): Laravel\\Lumen\\Application->handleShutdown()
#1 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 {main}
"} 
[2018-07-16 12:54:30] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:56:34] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:56:52] production.ERROR: ErrorException: Undefined index: get in /var/www/src/App/Application/Routing/RouteProvider.php:54
Stack trace:
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\Application\Routing\RouteProvider->handleRoutings(Object(Laravel\Lumen\Routing\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\Application\Routing\RouteProvider->register(Object(Laravel\Lumen\Routing\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\Service\Route\RouteProvider->register(Object(Xervice\Service\Lumen\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\Service\Bootstrap\ApplicationBootstrap->boot(Object(Xervice\Service\Lumen\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\Service\Application\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (ErrorException(code: 0): Undefined index: get at /var/www/src/App/Application/Routing/RouteProvider.php:54)
[stacktrace]
#0 /var/www/src/App/Application/Routing/RouteProvider.php(54): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(8, 'Undefined index...', '/var/www/src/Ap...', 54, Array)
#1 /var/www/src/App/Application/Routing/RouteProvider.php(32): App\\Application\\Routing\\RouteProvider->handleRoutings(Object(Laravel\\Lumen\\Routing\\Router), Array, 'get')
#2 /var/www/vendor/xervice/service/src/Xervice/Service/Route/RouteProvider.php(34): App\\Application\\Routing\\RouteProvider->register(Object(Laravel\\Lumen\\Routing\\Router))
#3 /var/www/vendor/xervice/service/src/Xervice/Service/Bootstrap/ApplicationBootstrap.php(44): Xervice\\Service\\Route\\RouteProvider->register(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(42): Xervice\\Service\\Bootstrap\\ApplicationBootstrap->boot(Object(Xervice\\Service\\Lumen\\ApplicationBridge))
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(24): Xervice\\Service\\Application\\Application->boot()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 12:58:08] production.ERROR: PDOException: could not find driver in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php(59): PDO->__construct('pgsql:host=127....', 'dockerci', 'dockerci', Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php(69): Propel\Runtime\Connection\PdoConnection->__construct('pgsql:host=127....', 'dockerci', 'dockerci', Array)
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#8 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#9 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#10 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#11 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#14 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#15 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#16 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#17 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#18 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#19 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#20 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#29 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#30 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#31 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#32 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#33 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#34 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#35 {main}

Next Propel\Runtime\Adapter\Exception\AdapterException: Unable to open PDO connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#6 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#8 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#9 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#12 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#15 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#16 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#17 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#18 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#29 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#30 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#31 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#32 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#33 {main}

Next Propel\Runtime\Connection\Exception\ConnectionException: Unable to open connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#13 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#32 {main} {"exception":"[object] (Propel\\Runtime\\Connection\\Exception\\ConnectionException(code: 0): Unable to open connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43, Propel\\Runtime\\Adapter\\Exception\\AdapterException(code: 0): Unable to open PDO connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72, PDOException(code: 0): could not find driver at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59)
[stacktrace]
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\\Runtime\\Connection\\ConnectionFactory::create(Array, Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getWriteConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getReadConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\\Runtime\\ServiceContainer\\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\\Runtime\\ActiveQuery\\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\\User\\Model\\User->checkUserExists(Object(DataProvider\\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\\User\\Model\\User->createUser(Object(DataProvider\\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\\User\\UserFacade->createUser(Object(DataProvider\\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\\User\\Controller\\UserController->registerAction(Object(DataProvider\\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\\Service\\Controller\\AbstractApiController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#13 [internal function]: App\\Application\\Controller\\MyAppController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\\Container\\BoundMethod::call(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\\Container\\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\\Lumen\\Application->callLumenController(Object(App\\User\\Controller\\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\\Lumen\\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#32 {main}
"} 
[2018-07-16 12:59:10] production.ERROR: PDOException: could not find driver in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php(59): PDO->__construct('pgsql:host=127....', 'dockerci', 'dockerci', Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php(69): Propel\Runtime\Connection\PdoConnection->__construct('pgsql:host=127....', 'dockerci', 'dockerci', Array)
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#8 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#9 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#10 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#11 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#14 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#15 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#16 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#17 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#18 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#19 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#20 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#29 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#30 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#31 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#32 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#33 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#34 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#35 {main}

Next Propel\Runtime\Adapter\Exception\AdapterException: Unable to open PDO connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#6 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#8 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#9 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#12 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#15 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#16 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#17 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#18 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#29 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#30 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#31 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#32 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#33 {main}

Next Propel\Runtime\Connection\Exception\ConnectionException: Unable to open connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#13 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#32 {main} {"exception":"[object] (Propel\\Runtime\\Connection\\Exception\\ConnectionException(code: 0): Unable to open connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43, Propel\\Runtime\\Adapter\\Exception\\AdapterException(code: 0): Unable to open PDO connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72, PDOException(code: 0): could not find driver at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59)
[stacktrace]
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\\Runtime\\Connection\\ConnectionFactory::create(Array, Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getWriteConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getReadConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\\Runtime\\ServiceContainer\\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\\Runtime\\ActiveQuery\\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\\User\\Model\\User->checkUserExists(Object(DataProvider\\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\\User\\Model\\User->createUser(Object(DataProvider\\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\\User\\UserFacade->createUser(Object(DataProvider\\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\\User\\Controller\\UserController->registerAction(Object(DataProvider\\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\\Service\\Controller\\AbstractApiController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#13 [internal function]: App\\Application\\Controller\\MyAppController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\\Container\\BoundMethod::call(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\\Container\\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\\Lumen\\Application->callLumenController(Object(App\\User\\Controller\\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\\Lumen\\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#32 {main}
"} 
[2018-07-16 12:59:41] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 13:00:10] production.ERROR: PDOException: could not find driver in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php(59): PDO->__construct('pgsql:host=appd...', 'dockerci', 'dockerci', Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php(69): Propel\Runtime\Connection\PdoConnection->__construct('pgsql:host=appd...', 'dockerci', 'dockerci', Array)
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#8 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#9 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#10 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#11 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#14 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#15 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#16 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#17 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#18 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#19 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#20 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#29 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#30 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#31 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#32 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#33 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#34 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#35 {main}

Next Propel\Runtime\Adapter\Exception\AdapterException: Unable to open PDO connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php(41): Propel\Runtime\Adapter\Pdo\PdoAdapter->getConnection(Array)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#5 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#6 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#7 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#8 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#9 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#12 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#13 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#15 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#16 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#17 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#18 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#28 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#29 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#30 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#31 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#32 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#33 {main}

Next Propel\Runtime\Connection\Exception\ConnectionException: Unable to open connection in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\Runtime\Connection\ConnectionFactory::create(Array, Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\Runtime\Connection\ConnectionManagerSingle->getWriteConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\Runtime\Connection\ConnectionManagerSingle->getReadConnection(Object(Propel\Runtime\Adapter\Pdo\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\Runtime\ServiceContainer\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#13 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#32 {main} {"exception":"[object] (Propel\\Runtime\\Connection\\Exception\\ConnectionException(code: 0): Unable to open connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionFactory.php:43, Propel\\Runtime\\Adapter\\Exception\\AdapterException(code: 0): Unable to open PDO connection at /var/www/vendor/propel/propel/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php:72, PDOException(code: 0): could not find driver at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/PdoConnection.php:59)
[stacktrace]
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(76): Propel\\Runtime\\Connection\\ConnectionFactory::create(Array, Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/ConnectionManagerSingle.php(90): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getWriteConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php(372): Propel\\Runtime\\Connection\\ConnectionManagerSingle->getReadConnection(Object(Propel\\Runtime\\Adapter\\Pdo\\PgsqlAdapter))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1018): Propel\\Runtime\\ServiceContainer\\StandardServiceContainer->getReadConnection('default')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOne(NULL)
#5 [internal function]: Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\\Runtime\\ActiveQuery\\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\\User\\Model\\User->checkUserExists(Object(DataProvider\\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\\User\\Model\\User->createUser(Object(DataProvider\\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\\User\\UserFacade->createUser(Object(DataProvider\\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\\User\\Controller\\UserController->registerAction(Object(DataProvider\\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\\Service\\Controller\\AbstractApiController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#13 [internal function]: App\\Application\\Controller\\MyAppController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\\Container\\BoundMethod::call(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\\Container\\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\\Lumen\\Application->callLumenController(Object(App\\User\\Controller\\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\\Lumen\\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#32 {main}
"} 
[2018-07-16 13:04:46] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 13:04:47] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 13:04:48] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 13:22:33] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
[2018-07-16 13:22:37] production.ERROR: PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "app_user" does not exist
LINE 1: ...rstname, app_user.lastname, app_user.company FROM app_user W...
                                                             ^ in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php:194
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php(194): PDOStatement->execute(NULL)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php(2642): Propel\Runtime\Connection\StatementWrapper->execute()
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\Runtime\ActiveQuery\Criteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\Runtime\ActiveQuery\ModelCriteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(Object(Propel\Runtime\Connection\ConnectionWrapper))
#5 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#13 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#32 {main}

Next Propel\Runtime\Exception\PropelException: Unable to execute SELECT statement [SELECT app_user.user_id, app_user.email, app_user.password, app_user.firstname, app_user.lastname, app_user.company FROM app_user WHERE app_user.email=:p1 LIMIT 1] in /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php:2648
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\Runtime\ActiveQuery\Criteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\Runtime\ActiveQuery\ModelCriteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(Object(Propel\Runtime\Connection\ConnectionWrapper))
#3 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#5 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#6 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#7 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#8 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#9 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#11 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#12 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#15 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#16 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#17 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#26 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#27 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#28 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#29 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#30 {main} {"exception":"[object] (Propel\\Runtime\\Exception\\PropelException(code: 0): Unable to execute SELECT statement [SELECT app_user.user_id, app_user.email, app_user.password, app_user.firstname, app_user.lastname, app_user.company FROM app_user WHERE app_user.email=:p1 LIMIT 1] at /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php:2648, PDOException(code: 42P01): SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"app_user\" does not exist
LINE 1: ...rstname, app_user.lastname, app_user.company FROM app_user W...
                                                             ^ at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php:194)
[stacktrace]
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\\Runtime\\ActiveQuery\\Criteria->doSelect(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\\Runtime\\ActiveQuery\\ModelCriteria->doSelect(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOne(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#3 [internal function]: Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#5 /var/www/src/App/User/Model/User.php(111): Propel\\Runtime\\ActiveQuery\\ModelCriteria->__call('findOneByEmail', Array)
#6 /var/www/src/App/User/Model/User.php(42): App\\User\\Model\\User->checkUserExists(Object(DataProvider\\UserDataProvider))
#7 /var/www/src/App/User/UserFacade.php(32): App\\User\\Model\\User->createUser(Object(DataProvider\\UserDataProvider))
#8 /var/www/src/App/User/Controller/UserController.php(26): App\\User\\UserFacade->createUser(Object(DataProvider\\UserDataProvider))
#9 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\\User\\Controller\\UserController->registerAction(Object(DataProvider\\UserDataProvider))
#10 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\\Service\\Controller\\AbstractApiController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#11 [internal function]: App\\Application\\Controller\\MyAppController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#12 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#14 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Object(Closure))
#15 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\\Container\\BoundMethod::call(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Array, NULL)
#16 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\\Container\\Container->call(Array, Array)
#17 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\\Lumen\\Application->callLumenController(Object(App\\User\\Controller\\UserController), 'apiAction', Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#26 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#27 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#28 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#29 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#30 {main}
"} 
[2018-07-16 13:23:15] production.ERROR: PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "app_user" does not exist
LINE 1: ...rstname, app_user.lastname, app_user.company FROM app_user W...
                                                             ^ in /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php:194
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php(194): PDOStatement->execute(NULL)
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php(2642): Propel\Runtime\Connection\StatementWrapper->execute()
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\Runtime\ActiveQuery\Criteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#3 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\Runtime\ActiveQuery\ModelCriteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(Object(Propel\Runtime\Connection\ConnectionWrapper))
#5 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#6 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#7 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#8 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#9 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#11 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#12 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#13 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#14 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#17 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#26 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#27 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#28 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#29 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#30 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#31 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#32 {main}

Next Propel\Runtime\Exception\PropelException: Unable to execute SELECT statement [SELECT app_user.user_id, app_user.email, app_user.password, app_user.firstname, app_user.lastname, app_user.company FROM app_user WHERE app_user.email=:p1 LIMIT 1] in /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php:2648
Stack trace:
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\Runtime\ActiveQuery\Criteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\Runtime\ActiveQuery\ModelCriteria->doSelect(Object(Propel\Runtime\Connection\ConnectionWrapper))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\Runtime\ActiveQuery\ModelCriteria->findOne(Object(Propel\Runtime\Connection\ConnectionWrapper))
#3 [internal function]: Propel\Runtime\ActiveQuery\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#5 /var/www/src/App/User/Model/User.php(111): Propel\Runtime\ActiveQuery\ModelCriteria->__call('findOneByEmail', Array)
#6 /var/www/src/App/User/Model/User.php(42): App\User\Model\User->checkUserExists(Object(DataProvider\UserDataProvider))
#7 /var/www/src/App/User/UserFacade.php(32): App\User\Model\User->createUser(Object(DataProvider\UserDataProvider))
#8 /var/www/src/App/User/Controller/UserController.php(26): App\User\UserFacade->createUser(Object(DataProvider\UserDataProvider))
#9 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\User\Controller\UserController->registerAction(Object(DataProvider\UserDataProvider))
#10 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\Service\Controller\AbstractApiController->apiAction(Object(Illuminate\Http\Request), 'register')
#11 [internal function]: App\Application\Controller\MyAppController->apiAction(Object(Illuminate\Http\Request), 'register')
#12 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Object(Closure))
#15 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\Container\BoundMethod::call(Object(Xervice\Service\Lumen\ApplicationBridge), Array, Array, NULL)
#16 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\Container\Container->call(Array, Array)
#17 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\Lumen\Application->callLumenController(Object(App\User\Controller\UserController), 'apiAction', Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\Lumen\Application->callControllerAction(Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#26 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#27 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#28 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#29 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#30 {main} {"exception":"[object] (Propel\\Runtime\\Exception\\PropelException(code: 0): Unable to execute SELECT statement [SELECT app_user.user_id, app_user.email, app_user.password, app_user.firstname, app_user.lastname, app_user.company FROM app_user WHERE app_user.email=:p1 LIMIT 1] at /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/Criteria.php:2648, PDOException(code: 42P01): SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"app_user\" does not exist
LINE 1: ...rstname, app_user.lastname, app_user.company FROM app_user W...
                                                             ^ at /var/www/vendor/propel/propel/src/Propel/Runtime/Connection/StatementWrapper.php:194)
[stacktrace]
#0 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1988): Propel\\Runtime\\ActiveQuery\\Criteria->doSelect(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#1 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1024): Propel\\Runtime\\ActiveQuery\\ModelCriteria->doSelect(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#2 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(1317): Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOne(Object(Propel\\Runtime\\Connection\\ConnectionWrapper))
#3 [internal function]: Propel\\Runtime\\ActiveQuery\\ModelCriteria->findOneBy('Email', 'contact@mibexx....')
#4 /var/www/vendor/propel/propel/src/Propel/Runtime/ActiveQuery/ModelCriteria.php(2194): call_user_func_array(Array, Array)
#5 /var/www/src/App/User/Model/User.php(111): Propel\\Runtime\\ActiveQuery\\ModelCriteria->__call('findOneByEmail', Array)
#6 /var/www/src/App/User/Model/User.php(42): App\\User\\Model\\User->checkUserExists(Object(DataProvider\\UserDataProvider))
#7 /var/www/src/App/User/UserFacade.php(32): App\\User\\Model\\User->createUser(Object(DataProvider\\UserDataProvider))
#8 /var/www/src/App/User/Controller/UserController.php(26): App\\User\\UserFacade->createUser(Object(DataProvider\\UserDataProvider))
#9 /var/www/vendor/xervice/service/src/Xervice/Service/Controller/AbstractApiController.php(59): App\\User\\Controller\\UserController->registerAction(Object(DataProvider\\UserDataProvider))
#10 /var/www/src/App/Application/Controller/MyAppController.php(25): Xervice\\Service\\Controller\\AbstractApiController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#11 [internal function]: App\\Application\\Controller\\MyAppController->apiAction(Object(Illuminate\\Http\\Request), 'register')
#12 /var/www/vendor/illuminate/container/BoundMethod.php(29): call_user_func_array(Array, Array)
#13 /var/www/vendor/illuminate/container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#14 /var/www/vendor/illuminate/container/BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Object(Closure))
#15 /var/www/vendor/illuminate/container/Container.php(564): Illuminate\\Container\\BoundMethod::call(Object(Xervice\\Service\\Lumen\\ApplicationBridge), Array, Array, NULL)
#16 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(373): Illuminate\\Container\\Container->call(Array, Array)
#17 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(339): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(313): Laravel\\Lumen\\Application->callLumenController(Object(App\\User\\Controller\\UserController), 'apiAction', Array)
#19 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(275): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(260): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(230): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#23 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#24 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#25 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#26 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#27 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#28 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#29 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#30 {main}
"} 
[2018-07-16 13:24:13] production.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\Lumen\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\Service\Application\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\Service\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\Application\Kernel\Kernel->run()
#8 {main} {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0):  at /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226)
[stacktrace]
#0 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#1 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}()
#2 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#3 /var/www/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\\Lumen\\Application->dispatch(NULL)
#4 /var/www/vendor/xervice/service/src/Xervice/Service/Application/Application.php(49): Laravel\\Lumen\\Application->run()
#5 /var/www/vendor/xervice/service/src/Xervice/Service/ServiceFacade.php(25): Xervice\\Service\\Application\\Application->run()
#6 /var/www/src/App/Application/Kernel/Kernel.php(46): Xervice\\Service\\ServiceFacade->startApplication()
#7 /var/www/html/index.php(13): App\\Application\\Kernel\\Kernel->run()
#8 {main}
"} 
